### ISSUE TITLE
Hard to reproduce pretrained model's results

### ISSUE THREAD

**Issue by dypromise** [royorel/Lifespan_Age_Transformation_Synthesis#7]
Hi, royorel!
Thank you for sharing the excellent work! I want to reproduce the pretrained model's result, I encountered some issues. I use FFHQ 1024x1024 data, aligned  by the method in 'download_ffhq_aging' script, and produced parsings image by your deeplab script. I train the model with female data use parsing to cover background, but cant converge a bit before 50epoch. loss_D_real and loss_D_fake decreasing to 0 fast. what do you think the reason? Hope for your suggestions.
the loss and training image:
![image](https://user-images.githubusercontent.com/21032349/93069129-dd41ad00-f6af-11ea-9c5a-c1d6a3b1822d.png)
![image](https://user-images.githubusercontent.com/21032349/93069616-77a1f080-f6b0-11ea-8e87-bf033c3585a7.png)

**Comment by royorel** [comment]
This can happen sometimes... Your model will eventually converge as the learning rate decays (after epochs 50 & 100 in the default settings). It happens because the initial learning rate is relatively high, but that high initial learning rate is needed to get good final results. 

You can try to decay the learning rate earlier (like in epochs 25 & 75) to make the model converge faster
